installing/installing_aws/installing-aws-wavelength.adoc

```diff
@@ -0,0 +1,280 @@
+or create your own template.
+====
+endif::aws-localzones[]
+
+[IMPORTANT]
+====
+The steps for performing an installer-provisioned infrastructure installation are provided for example purposes only.
+Installing a cluster in an existing VPC requires that you have knowledge of the cloud provider and the
+installation process of {product-title}. You can use a CloudFormation template to assist you with completing
+these steps or to help model your own cluster installation. Instead of using the CloudFormation template to create
+resources, you can decide to use other methods for generating these resources.
+====
+
+// Creating a VPC in AWS
+include::modules/installation-creating-aws-vpc-localzone.adoc[leveloffset=+2]
+
+// CloudFormation template for the VPC
+include::modules/installation-cloudformation-vpc-localzone.adoc[leveloffset=+2]
+
+ifdef::aws-localzones[]
+// AWS security groups
+// Do we need this for LZ as the example is already provided in the section "Edge compute pools and AWS Local Zones"?
+include::modules/installation-aws-security-groups.adoc[leveloffset=+2]
+
+// CloududFormation template for the subnet for {aws-service-name-full}
+include::modules/installation-cloudformation-subnet-localzone.adoc[leveloffset=+2]
+
+endif::aws-localzones[]
+ifdef::aws-wavelength[]
+
+// Creating a Carrier Gateway in AWS VPC Carrier Gateway
+include::modules/installation-creating-aws-vpc-carrier-gw.adoc[leveloffset=+2]
+
+// CloudFormation template for the VPC Carrier Gateway
+include::modules/installation-cloudformation-vpc-carrier-gw.adoc[leveloffset=+2]
+
+// Creating the subnets in AWS {aws-service-name-full}
+include::modules/installation-creating-aws-vpc-subnets.adoc[leveloffset=+2]
+
+// CloudFormation template for the VPC Subnets
+include::modules/installation-cloudformation-vpc-subnets.adoc[leveloffset=+2]
+
+endif::aws-wavelength[]
+
+[role="_additional-resources"]
+.Additional resources
+
+* You can view details about the CloudFormation stacks that you create by navigating to the
+  link:https://console.aws.amazon.com/cloudformation/[AWS CloudFormation console].
+
+// Modifying an installation configuration file to use AWS {aws-service-name-full} subnets
+include::modules/installing-aws-edge-zones-custom-vpc-config.adoc[leveloffset=+2]
+
+[role="_additional-resources"]
+.Additional resources
+
+* See link:https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html[Configuration and credential file settings]
+  in the AWS documentation for more information about AWS profile and credential configuration.
+
+.Next steps
+* xref:../../installing/installing_aws/installing-aws-localzone.adoc#installation-launching-installer_installing-aws-localzone[Deploying the cluster]
+
+// Creating edge worker nodes in public subnets (optional)
+include::modules/machineset-aws-edge-zone-public.adoc[leveloffset=+1]
+
+//TODO: Put this back if QE validates it:
```

Comment:
Can I remove these entries from PR or should I include them as commented out entries for the purpose of history tracking?